client/src/app/site/users/models/view-user.ts

```diff
@@ -68,6 +68,10 @@ export class ViewUser extends BaseProjectableViewModel<User> implements Searchab
         return DelegationType.Neither;
     }
 
+    public get meeting_ids(): Id[] {
+        return this.user.meeting_ids;
+    }
+
```

Comment:
Is it necessary, that a `ViewUser` has this property? Usually, properties of a `User` are treated as properties of a `ViewUser`, too.

Reply:
more or less. The pseudo-automatic transaction from properties from `User` into `ViewUser` has issues that prevent automatic testing quite heavily, thus declaring is implicit seems a lot safer. There might be other solutions on how ViewModels were created.